README.md

```diff
@@ -2,6 +2,7 @@
 Stand-alone add-on for use in Shield studies.
 
 #To build
+`sass data/recommendation/style.scss data/recommendation/style.css --sourcemap=none`
 `$ jpm xpi`
```

Comment:
This renders really badly, since in Markdown, adjacent lines are considered one paragraph and can be reflowed together. You have a few options here:

* Make this two paragraphs, like this:
  ```
  # To build
  `$ sass data/recommendation/style.scss data/recommendation/style.css --sourcemap=none`

  `$ jpm xpi`
  ```
* Make it into a bulleted list, like this:

  ```
  # To build
  * `$ sass data/recommendation/style.scss data/recommendation/style.css --sourcemap=none`
  * `$ jpm xpi`
  ```
* Use a code block

  ```
  # To build

  ```bash
  $ sass data/recommendation/style.scss data/recommendation/style.css --sourcemap=none
  $ jpm xpi
  `` `
  ```